src/app/shared/components/template/services/instance/template-row.service.ts

```diff
@@ -195,7 +195,12 @@ export class TemplateRowService extends TemplateInstanceService {
       const processed = await this.processSingleRow(preProcessedRow, isNestedTemplate);
       // only include rows that do not return undefined (e.g. set_variable)
       if (processed) {
-        processedRows.push(processed);
```

Comment:
minor change required to allow for processing that returns more than one row for each input. Could be useful in the future if planning to render nested templates in the same way (currently render in containers but might be nicer if all components sit at same level as more akin to how they were designed in excel)